tools/sistr_cmd/sistr_cmd.xml

```diff
@@ -0,0 +1,227 @@
+<tool id="sistr_cmd" name="sistr_cmd" version="0.3.1">
+  <description>
+    Salmonella In Silico Typing Resource commandline tool for serovar prediction
+  </description>
+  <requirements>
+    <requirement type="package" version="0.3.1">sistr_cmd</requirement>
+  </requirements>
+  <stdio>
+    <exit_code range="1:" />
+  </stdio>
+  <command><![CDATA[
+  sistr 
+    -f $output_format
+    #if $output_format == "csv"
+      -o sistr-report.csv
+    #elif $output_format == "json"
+      -o sistr-report.json
+    #end if
+    -p $cgmlst_profiles
+    -n $novel_alleles
+    -a $alleles_output
+    -T $tmp_dir
+    #if $keep_tmp
+      -K
+    #end if
+    #if $use_full_cgmlst_db
+      --use-full-cgmlst-db
+    #end if
+    #if $no_cgmlst
+      --no-cgmlst
+    #end if
+    #if $run_mash
+      -m
+    #end if
+    #if $qc
+      --qc
+    #end if
+    #if $threads > 1
+      -t $threads
+    #end if
+    #if $verbosity == 1
```

Comment:
The verbosity of the log messages defaults to ``INFO`` right now and setting it to ``-vvv`` results in a crash in v0.3.3 and earlier versions. I've fixed this in https://github.com/peterk87/sistr_cmd/commit/f096b5d2944dc19de8aa79696c137fe3f45927da and will release an updated version. I've commented out the input param for controlling logging verbosity for now.